apps/bot/setup-ec2.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# === Dowload all system dependencies ===
+sudo apt update -y
+sudo apt install -y git ffmpeg pulseaudio curl nodejs npm
+\ libatk-bridge2.0-0 libgtk-3-0 libappindicator3-1 libnss3 libxss1
+\ libasound2 libgbm1 libx11-xcb1 libxcomposite1 libxkbcommon0
+
+# === Setup Pulse Audio with virtual mic ===
+pulseaudio --start
+
+pactl load-module module-null-sink sink_name=virtual_sink sink_properties=device.description=Virtual_Sink
+pactl load-module module-loopback source=virtual_sink.monitor
+
+# === Clone project ===
+cd /home/ubuntu
+git clone --branch 44-feat-record-audio-from-ec2 https://github.com/BinaryStudioAcademy/bsa-2025-meetlytic.git
```

Comment:
If we are in this file then we have already cloned the project, no need to clone again